src/github/client/github-client.ts

```diff
@@ -135,11 +149,24 @@ export default class GitHubClient {
 	 * Get a single commit for the given repository.
 	 */
 	public getCommit = async (owner: string, repo: string, ref: string): Promise<AxiosResponse<Octokit.ReposGetCommitResponse>> => {
-		return await this.get<Octokit.ReposGetCommitResponse>(`/repos/:owner/:repo/commits/:ref`,  {}, {
+		return await this.get<Octokit.ReposGetCommitResponse>(`/repos/:owner/:repo/commits/:ref`, {}, {
 			owner,
 			repo,
 			ref
 		});
 	}
 
+	public async getNumberOfReposForInstallation(): Promise<number> {
+		const response = await this.graphql(`
+			query {
+				viewer {
+					repositories {
+						totalCount
+					}
+				}
+			}`)
+
+		return response?.data?.data?.viewer?.repositories?.totalCount as number;
```

Comment:
There was an error here, apparently resoponse.data is the response body and "data" is its top element

Reply:
🤦🏼